src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObj = sourceString
+    .split(';')
+    .map((itemFromArr) => itemFromArr.trim())
+    .filter((itemFromArr) => itemFromArr.length > 0)
```

Comment:
When chaining methods, it's best practice to have each method on a new line with the same indentation to improve code readability.